### PR TITLE
[WIP]Added comments for exported functions.

### PR DESCRIPTION
**Pull request by AndreaM16**
**What this PR does / why we need it**:
Fixing `golint` issues in `maya/pkg/msg/v1alpha1/msg.go` a request in [issue #2131](https://github.com/openebs/openebs/issues/2131)

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

`fixes#2131` opened in openebs.

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by AndreaM16** [comment]
There's some issue with DCO.

**Comment by sonasingh46** [comment]
@AndreaM16 
Thanks for the PR.
Whenever you commit you have to do `git commit -s`.
Post question on our slack channel #hacktoberfest2018 and learn more.
Join slack channel at `slack.openebs.io`

File: pkg/msg/v1alpha1/msg.go [hunk 1]
```diff
@@ -61,38 +66,44 @@ func (m *msg) GoString() string {
 // msgPredicate abstracts evaluation of a message condition
 type msgPredicate func(given *msg) bool
 
+// IsInfo returns true if given message's MType is InfoMsg
```
Comment (princerachit) [inline pkg/msg/v1alpha1/msg.go]: How about changing all the comments to `xxx returns true if given message is of type xxx`

Comment (AndreaM16) [reply pkg/msg/v1alpha1/msg.go]: Agreed.

**Comment by sonasingh46** [comment]
@AndreaM16 
Can you please fix the DCO issue. Looks good to me. Will wait for one more approval after DCO is fixed.

**Comment by sonasingh46** [comment]
@AndreaM16 
Closing this PR due to inactivity. Please let us know of we want to work on the same.

**Pull request closed without merging**